Ginger/Ginger/ApplicationModelsLib/POMModels/POMWizardLib/POMLearnConfigWizardPage.xaml.cs

```diff
@@ -128,7 +141,34 @@ private void SetAutoMapElementTypes()
             }
         }
 
+        private void SetAutoMapElementLocators()
+        {
+            if (mWizard.AutoMapElementLocatorsList.Count == 0)
+            {
+                switch (mAppPlatform)
+                {
+                    case ePlatformType.Web:
+                        foreach (eLocateBy locator in new WebPlatform().GetPlatformUIElementLocatorsList().ToList())
+                        {
+                            switch (locator)
+                            {
+                                case eLocateBy.ByID:
+                                case eLocateBy.ByName:
+                                case eLocateBy.ByRelXPath:
+                                    mWizard.AutoMapElementLocatorsList.Add(new ElementLocator() { Active = true, LocateBy = locator, Help = "Very Recommended(usually unique)" });
+
```

Comment:
we need to make sure that defualt Locators order, matches to the Order SelenuimDriver is going to add in learning phase.
so instead switch case on full list and pulling some locators here. add function to IWindowExplorer interface  which called "GetLearningLocators()" and selenuimDriver will implement it and return you the Locators types it supports, in correct order and with Help string.

Reply:
Done. But only for SeleniumDriver as currently, POM isn't supported for other platforms other than Web.